src/geometric_algebra/nursery/talk.lean

```diff
@@ -86,49 +94,54 @@ variables (K : Type u) [field K]
 
 variables (V : Type u) [add_comm_group V] [vector_space K V]
 
-structure GA
-(G : Type u)
-[ring G] extends algebra K G :=
+#print linear_map
+
+def sqr {α : Type u} [has_mul α] (x : α) := x*x
+local postfix `²`:80 := sqr
```

Comment:
This avoids the nasty lambda functions in the expressions, and shows `²` in the tactic state for as long as possible.